editor/project_manager.cpp

```diff
@@ -475,6 +475,13 @@ class ProjectDialog : public ConfirmationDialog {
 					}
 					ProjectSettings::CustomMap initial_settings;
 					initial_settings["rendering/vulkan/rendering/back_end"] = rasterizer_button_group->get_pressed_button()->get_meta(SNAME("driver_name"));
+					if (rasterizer_button_group->get_pressed_button()->get_meta("driver_name") == "Vulkan") {
+						initial_settings["rendering/driver/driver_name"] = "Vulkan";
+					} else {
+						initial_settings["rendering/driver/driver_name"] = "OpenGL3";
+						initial_settings["rendering/textures/vram_compression/import_etc2"] = false;
+						initial_settings["rendering/textures/vram_compression/import_etc"] = true;
+					}
```

Comment:
I wonder if this is what is causing #54488 It looks like maybe "driver_name" does not equal "Vulkan" Maybe it should be lowercase?

Reply:
Indeed, this is likely what's causing the issue. I noticed that we're going for lowercase driver names now, but I forgot to update that part of the code.

Eventually, we may want to do the same for audio drivers.

Reply:
See #54488 "driver_name" is actually an int. I recommend we just remove this block altogether until we are ready to expose opengl3 to the project_manager

Reply:
https://github.com/godotengine/godot/pull/54491